dc-cudami-editor/src/api.js

```diff
@@ -211,21 +211,16 @@ export async function saveFileResource(contextPath, fileResource) {
   const savedFileResource = await saveIdentifiable(
     contextPath,
     fileResource,
-    null,
-    null,
-    'fileResource',
-    false
+    'fileResource'
   )
   return savedFileResource
```

Comment:
Könnte man das nicht in einer Zeile zusammenfassen? ;-)

Reply:
Ja, ist umgesetzt :+1: :smile: